README.md

```diff
@@ -37,21 +40,78 @@ For Single Cell Expression Atlas run:
 create-bioentities-suggesters-scxa.sh
 ```
 
-## Load data
+## Load data - web cli - current
+
+This approach uses the atlas-web-cli JAR through conda, and is the most current setup. It first entails the creation of bioentities JSONL for a species and then loading it. This means that the process of creation of JSONLs can be distributed as much as possible, while queuing separately the later loading to avoid very high loads on the solr server. This unfortunately requires the setup of a large amount of environment variables for the application context of the web application. Note that this is not going through the actual Tomcat process of the web application, but through an independent execution (on each run) of the JAR file that contains the same code.
+
+```bash
+# here we show tests settings
+export ZK_HOST=${ZK_HOST:localhost}
+export ZK_PORT=${ZK_PORT:-2181}
+export SOLR_HOST=my_solr:8983
+export BIOENTITIES=$DIR/fixtures/
+export EXPERIMENT_FILES=$DIR/fixtures/experiment_files
+export jdbc_url="jdbc:postgresql://$POSTGRES_HOST:$POSTGRES_PORT/$POSTGRES_DB"
+export jdbc_username=$POSTGRES_USER
+export jdbc_password=$POSTGRES_PASSWORD
+export server_port=8081 #fake
+
+export output_dir=$( pwd )
+
+# create JSONL
+./bin/create_bioentities_jsonl.sh
+# Load
+export SPECIES=homo_sapiens
+export BIOENTITIES_JSONL_PATH=$( pwd )
+
+./bin/index_organism_annotations.sh
+```
+
+## Create mapping for analytics indexing
+
+As in the previous step, this also uses the atlas-web-cli. It will take files in the bioentity_properties and experiment_files expected by the wen application and the data in the currently loaded bioentities in solr
+to create mapping files from the expression data to the bioentities metadata.
+
+```bash
+# here we show tests settings
+export ZK_HOST=${ZK_HOST:localhost}
+export ZK_PORT=${ZK_PORT:-2181}
+export SOLR_HOST=my_solr:8983
+export BIOENTITIES=$DIR/fixtures/
+export EXPERIMENT_FILES=$DIR/fixtures/experiment_files
+export jdbc_url="jdbc:postgresql://$POSTGRES_HOST:$POSTGRES_PORT/$POSTGRES_DB"
+export jdbc_username=$POSTGRES_USER
+export jdbc_password=$POSTGRES_PASSWORD
+export server_port=8081 #fake
+
+export ACCESSIONS=E-MTAB-4754
+export CONDA_PREFIX=/opt/conda
+export output_dir=$( pwd )
+
+./bin/create_bioentities_property_map.sh
+```
+
+this will create in this case $( pwd )/homo_sapiens.map.bin for E-MTAB-4754 which can then be used by software in the index-analytics repo. Note that multiple accessions can be given here. `create_bioentities_property_map.sh` can be used as well for a complete species (and then the Java app will traverse all experiments in the file system and create the mappings for those in that species).
```

Comment:
"in the index-analytics repo" -> "in the index-analytics (gxa or scxa) repos"

Reply:
Actually this only applies to bulk, in scxa we don't put the expression content on the index.